src/components/__tests__/ThoughtCreate.js

```diff
@@ -0,0 +1,6 @@
+it('create thought', async () => {
+  const thought = document.wrapper.find(
+    'li.leaf div.thought-container div.thought div.editable',
```

Comment:
Based on my tests, `document.wrapper.find('div.editable')` works fine.

`.editable` does not work because there is both a `<ContentEditable className='editable'>` and a `<div className='editable'>`